kong/templates/nginx_kong.lua

```diff
@@ -29,11 +29,13 @@ lua_code_cache ${{LUA_CODE_CACHE}};
 lua_socket_pool_size ${{LUA_SOCKET_POOL_SIZE}};
 lua_max_running_timers 4096;
 lua_max_pending_timers 16384;
-lua_shared_dict kong 4m;
-lua_shared_dict cache ${{MEM_CACHE_SIZE}};
-lua_shared_dict process_events 1m;
-lua_shared_dict cluster_events 1m;
-lua_shared_dict cassandra 5m;
+lua_shared_dict kong                4m;
+lua_shared_dict kong_cache          ${{MEM_CACHE_SIZE}};
+lua_shared_dict kong_process_events 1m;
+lua_shared_dict kong_cluster_events 1m;
+> if database == "cassandra" then
+lua_shared_dict kong_cassandra      5m;
+> end
```

Comment:
On mobile, but this doesn't seem to reflect the last commit where all sizes were set to 5 m?

That said, any rationale to the 5m sizes?